<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Statistics</class>
 <widget class="QWidget" name="Statistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="bestScoreLbl">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>20</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Century Gothic&quot;;
color: rgb(43, 255, 20);</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="startedAtLbl_9">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>180</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);
font: 75 15pt &quot;Century Gothic&quot;;
</string>
   </property>
   <property name="text">
    <string>Lowest Score</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="startedAtLbl_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);
font: 75 15pt &quot;Century Gothic&quot;;

</string>
   </property>
   <property name="text">
    <string>Longest Done Interval :</string>
   </property>
  </widget>
  <widget class="QLabel" name="highestInWeekLbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;Century Gothic&quot;;
color: rgb(43, 255, 20);</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="startedAtLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>72</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);

</string>
   </property>
   <property name="text">
    <string>Started at :</string>
   </property>
  </widget>
  <widget class="QLabel" name="startedAtLbl_5">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>590</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);
font: 75 15pt &quot;Century Gothic&quot;;


</string>
   </property>
   <property name="text">
    <string>Longest Undone Interval :</string>
   </property>
  </widget>
  <widget class="QLabel" name="startDateLbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/StatisticsText.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="highestIn7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);
font: 75 15pt &quot;Century Gothic&quot;;

</string>
   </property>
   <property name="text">
    <string>Highest Score In The last 7 Days :</string>
   </property>
  </widget>
  <widget class="QLabel" name="bestIntervalLbl">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>590</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;Century Gothic&quot;;
color: rgb(43, 255, 20);</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="currentScoreLbl">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>130</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>20</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Century Gothic&quot;;
color: rgb(255, 170, 0);</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="titleLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>19</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);

</string>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="startedAtLbl_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>180</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);
font: 75 15pt &quot;Century Gothic&quot;;</string>
   </property>
   <property name="text">
    <string>Current Score</string>
   </property>
  </widget>
  <widget class="QLabel" name="lowestInWeekLbl">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;Century Gothic&quot;;
color: rgb(255, 6, 10);</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="worstScoreLbl">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>130</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>20</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;Century Gothic&quot;;
color: rgb(255, 6, 10);</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="highestInMonthLbl">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>320</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;Century Gothic&quot;;
color: rgb(43, 255, 20);</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="worstInterval">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>590</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;Century Gothic&quot;;
color: rgb(255, 6, 10);</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="startedAtLbl_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);
font: 75 15pt &quot;Century Gothic&quot;;
</string>
   </property>
   <property name="text">
    <string>Highest Score</string>
   </property>
  </widget>
  <widget class="QLabel" name="window">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1001</width>
     <height>641</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/statisticsWindow.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="closeB">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>0</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/close.png</normaloff>:/res/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="minimizeB">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>0</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/minimize.png</normaloff>:/res/minimize.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/version.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="startedAtLbl_10">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>590</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);
font: 75 15pt &quot;Century Gothic&quot;;

</string>
   </property>
   <property name="text">
    <string>Days</string>
   </property>
  </widget>
  <widget class="QLabel" name="startedAtLbl_11">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>590</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);
font: 75 15pt &quot;Century Gothic&quot;;

</string>
   </property>
   <property name="text">
    <string>Days</string>
   </property>
  </widget>
  <widget class="QLabel" name="highestIn30">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);
font: 75 15pt &quot;Century Gothic&quot;;

</string>
   </property>
   <property name="text">
    <string>Highest Score In The last 30 Days :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lowestIn30">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>400</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);
font: 75 15pt &quot;Century Gothic&quot;;

</string>
   </property>
   <property name="text">
    <string>Lowest Score In The last 30 Days :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lowestIn7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);
font: 75 15pt &quot;Century Gothic&quot;;

</string>
   </property>
   <property name="text">
    <string>Lowest Score In The last 7 Days :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lowestInMonthLbl">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;Century Gothic&quot;;
color: rgb(255, 6, 10);</string>
   </property>
   <property name="text">
    <string>12</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>90</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(243, 243, 243);
font: 75 13pt &quot;Century Gothic&quot;;
</string>
   </property>
   <property name="text">
    <string>Statistics Scope :</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>580</y>
     <width>20</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="scope">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>90</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>13</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(62, 62, 62);
color: rgb(243, 243, 243);
font: 75 13pt &quot;Century Gothic&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>Last 7 Days</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Last 30 Days</string>
    </property>
   </item>
  </widget>
  <widget class="QCustomPlot" name="plotting" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>651</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <zorder>window</zorder>
  <zorder>bestScoreLbl</zorder>
  <zorder>startedAtLbl_9</zorder>
  <zorder>startedAtLbl_6</zorder>
  <zorder>highestInWeekLbl</zorder>
  <zorder>startedAtLbl</zorder>
  <zorder>startedAtLbl_5</zorder>
  <zorder>startDateLbl</zorder>
  <zorder>highestIn7</zorder>
  <zorder>bestIntervalLbl</zorder>
  <zorder>currentScoreLbl</zorder>
  <zorder>startedAtLbl_3</zorder>
  <zorder>lowestInWeekLbl</zorder>
  <zorder>worstScoreLbl</zorder>
  <zorder>highestInMonthLbl</zorder>
  <zorder>worstInterval</zorder>
  <zorder>startedAtLbl_4</zorder>
  <zorder>titleLbl</zorder>
  <zorder>closeB</zorder>
  <zorder>minimizeB</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>startedAtLbl_10</zorder>
  <zorder>startedAtLbl_11</zorder>
  <zorder>highestIn30</zorder>
  <zorder>lowestIn30</zorder>
  <zorder>lowestIn7</zorder>
  <zorder>lowestInMonthLbl</zorder>
  <zorder>label_3</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>scope</zorder>
  <zorder>plotting</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
